cmd/pull.go

```diff
@@ -27,22 +27,28 @@ var cloud_engine string
 var output_dir string
 var play string
 var verbose bool
+var extraVars map[string]string
 
 var pullCmd = &cobra.Command{
 	Use:   "pull",
 	Short: "Pull configurations from Podman or OCP",
 	Long: `This command pulls configuration files by services from Podman
 	environment or OCP. For example:
-  os-diff pull --cloud_engine=ocp --inventory=$PWD/hosts --output-dir=/tmp`,
+    os-diff pull --cloud_engine=ocp --inventory=$PWD/hosts --output-dir=/tmp`,
 	Run: func(cmd *cobra.Command, args []string) {
 
 		ansiblePlaybookConnectionOptions := &ansible.AnsiblePlaybookConnectionOptions{
 			Connection: "local",
 		}
 
+		envMap := make(map[string]interface{})
```

Comment:
Is this the best variable name for this?  Being verbose is better, I think?

Reply:
ack I changed it in the next commit